### PR TITLE
feature: add scripts/win_rate_plot.py

### PR DESCRIPTION
**Pull request by YuriCat**
added a caution.

### CODE REVIEW

File: scripts/win_rate_plot.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+fig = plt.figure()
+ax = fig.add_subplot(1, 1, 1)
+
+last_win_rate = {}
+for opponent in opponents:
+    if opponent == 'total':
+        continue
+    wp_list = averaged_wp_lists[opponent]
+    start = start_epoch[opponent]
+    # ax.plot(clipped_epoch_list[start:], wp_list[start:], label=opponent)
```
Comment (ikki407) [inline scripts/win_rate_plot.py:91]: this line should be removed (duplicated below)

**Comment by ikki407** [comment]
If this script will be merged, should we add a new function to save a log file using `logging` module? (e.g. `log_dir: 'logs' in config.yaml`) What do you think?
Another way, we can show a sample code to log stdout/stderr with shell command such as `script`, `tee`, ...

File: scripts/win_rate_plot.py (hunk 1 above)
Comment (YuriCat) [reply scripts/win_rate_plot.py:91]: This line is prepared in order to show the WR curve with epochs.
However, we should also change x-axis label.

Comment (ikki407) [reply scripts/win_rate_plot.py:91]: Sorry!! My mistake...

**Pull request merged**